app/models/initiative.rb

```diff
@@ -43,5 +45,19 @@ class Initiative < ApplicationRecord
   has_one :parent, class_name: "Initiative", foreign_key: "parent_id"
 
   validates :source, presence: true
-  validates :title, presence: true
+  validates :title, presence: true, uniqueness: true
+  validate :must_have_manager
+
+  def must_have_manager
+    if not (
+        source.is_a?(Manager) or
+        helpers.any? { |h| h.is_a?(Manager) }
+      )
+        errors.add :wyeworker_initiative_belongings, "An initiative must have a manager involved, as a source or as a helper"
+    end
+  end
+
+  before_create do
+    self.description = self.description || "No description"
```

Comment:
There is a ruby syntax sugar that you can use here, instead of writing this you can just use:
self.description ||= "No description"
😄